packages/puppeteer-core/src/node/ScreenRecorder.ts

```diff
@@ -174,30 +221,60 @@ export class ScreenRecorder extends PassThrough {
     );
   }
 
-  #getFormatArgs(format: 'webm' | 'gif') {
+  #getFormatArgs(
+    format: FileFormat,
+    fps: number | 'source_fps',
+    loop: number,
+    delay: number,
+    quality: number,
+    colors: number,
+  ) {
+    const libvpx = [
+      // Sets the codec to use.
+      ['-c:v', 'vp9'],
+      // Sets the quality. Lower the better.
+      ['-crf', `${quality}`],
+      // Sets the quality and how efficient the compression will be.
+      ['-deadline', 'realtime', '-cpu-used', `${os.cpus().length / 2}`],
```

Comment:
Funny bug its max value here is 8 - https://trac.ffmpeg.org/wiki/Encode/VP9#CPUUtilizationSpeed
```suggestion
      ['-deadline', 'realtime', '-cpu-used', `${Math.min(os.cpus().length / 2, 8)}`],
```